boards/xtensa/qemu_xtensa/doc/board.rst

```diff
@@ -0,0 +1,45 @@
+.. _qemu_xtensa:
+
+Xtensa Emulation (QEMU)
+#######################
+
+Overview
+********
+
+The QEMU board configuration is used to emulate the Xtensa architecture. This board
+configuration provides support for the Xtensa simulation environment.
+
+Hardware
+********
+
+Programming and Debugging
+*************************
+
+Use this board configuration to run basic Zephyr applications and kernel tests
+in an emulated enviroments. For example:
+
+.. code-block:: console
+
+        $ make -C samples/synchronization BOARD=qemu_xtensa run
+
+This will build an image and boots it using QEMU with the following output:
```

Comment:
This will build an image with the synchronization sample app, boot it using QEMU,
    and display the following console output: